src/v/redpanda/admin/api-doc/partition.json

```diff
@@ -210,6 +244,84 @@
                     "description": "Replica assignments"
                 }
             }
+        },
+        "producer_identity": {
+            "id": "producer_identity",
+            "description": "Producer identity",
+            "properties": {
+                "id": {
+                    "type": "long",
+                    "description": "Producer id"
+                },
+                "epoch": {
+                    "type": "long",
+                    "description": "Producer epoch"
+                }
+            }
+        },
+        "active_transaction": {
+            "id": "active_transaction",
+            "description": "Active transactions for current partition",
+            "properties": {
+                "producer_id": {
+                    "type": "producer_identity",
+                    "description": "Producer id"
+                },
+                "first_offset": {
+                    "type": "long",
+                    "description": "First offset"
+                },
+                "last_update": {
```

Comment:
let's use duration since the last update then we don't need to deal with fmt and it's easier to process this response from a script; for example we my name it `staleness_ms`